src/pages/security/network/network.ts

```diff
@@ -0,0 +1,98 @@
+import { Component } from '@angular/core';
+import { ConfigurationHelper } from '@aerogear/core';
+import { AuthService } from '@aerogear/auth';
+import { authProvider } from '../../../services/auth.service';
+import { ToastController } from 'ionic-angular';
+import { NavController } from 'ionic-angular';
+import { LoadingController } from 'ionic-angular';
+
+import { Http } from '@angular/http';
+
+import {RequestOptions} from '@angular/http';
+import {Headers} from '@angular/http';
+import 'rxjs/add/operator/map';
+declare var require: any
+var config = require('../../../mobile-services.json');
+declare let window: any;
+
+@Component({
+  selector: 'page-network',
+  templateUrl: 'network.html',
+  providers: [authProvider]
+})
+export class NetworkPage {
+  apiAccessRole: string;
+  progress: number;
+  headerConfig: object;
+  apiServerUrl: string;
+  apiEndpoint: string;
+  pinningSuccess: boolean;
+  responseRecieved: boolean;
+  requestSuccess: boolean;
+  requestFailure: boolean;
+  configuration: any;
+
+  constructor(public loadingCtrl: LoadingController, public toastCtrl: ToastController, private auth: AuthService, public navCtrl: NavController, public http: Http) {
+    this.configuration = new ConfigurationHelper(config).getConfig(AuthService.ID);
```

Comment:
This element is internal part of the SDK. We going to remove it from imports now.